jetty-slf4j-impl/src/main/java/org/eclipse/jetty/logging/StdErrAppender.java

```diff
@@ -0,0 +1,331 @@
+    }
+
+    public void setStream(PrintStream stream)
+    {
+        this.stderr = stream;
+    }
+
+    private void format(StringBuilder builder, JettyLogger logger, Level level, long timestamp, String threadName, Throwable throwable, String message, Object... argumentArray)
+    {
+        Throwable cause = throwable;
+
+        // Timestamp
+        ZonedDateTime tsInstant = Instant.ofEpochMilli(timestamp).atZone(timezone);
+        timestampFormatter.formatTo(tsInstant, builder);
+
+        // Level
+        builder.append(':').append(renderedLevel(level));
+
+        // Logger Name
+        builder.append(':');
+        if (condensedNames)
+        {
+            builder.append(logger.getCondensedName());
+        }
+        else
+        {
+            builder.append(logger.getName());
+        }
+
+        // Thread Name
+        builder.append(':');
+        builder.append(threadName); // TODO: support TAG_PAD configuration
+        builder.append(':');
+
+        // Message
+        builder.append(' ');
+
+        if (strictFormat)
+        {
+            FormattingTuple ft = MessageFormatter.arrayFormat(message, argumentArray);
+            appendEscaped(builder, ft.getMessage());
+            if (cause == null)
+            {
+                cause = ft.getThrowable();
+            }
+        }
+        else
+        {
+            // TODO: this should really be removed, as it violates the slf4j API contract for throwables and such
+            StringBuilder msg = new StringBuilder();
+            Object[] args = argumentArray == null ? EMPTY_ARGS : argumentArray;
+            msg.append(Objects.requireNonNullElseGet(message, () -> "{} ".repeat(args.length)));
+            String braces = "{}";
+            int start = 0;
+            for (Object arg : args)
+            {
+                int bracesIndex = msg.indexOf(braces, start);
+                if (bracesIndex < 0)
+                {
+                    appendEscaped(builder, msg.substring(start));
+                    builder.append(" ");
+                    if (arg != null)
+                        builder.append(arg);
+                    start = msg.length();
+                }
+                else
+                {
+                    appendEscaped(builder, msg.substring(start, bracesIndex));
+                    builder.append(arg);
+                    start = bracesIndex + braces.length();
+                }
+            }
+            appendEscaped(builder, msg.substring(start));
+        }
+
+        // Throwable
+        if (cause != null)
+        {
+            if (logger.isHideStacks())
+            {
+                builder.append(": ").append(cause);
+            }
+            else
+            {
+                appendCause(builder, cause, "");
+            }
+        }
+    }
+
+    private String renderedLevel(Level level)
+    {
+        switch (level)
+        {
+            case ERROR:  // New for Jetty 10+
+                return "ERROR";
```

Comment:
The intention is that all the level tags are the same length so that they read nicely down the screen (not sure how ignored escaped that), so they should either be:
```
"ERR "
"WARN"
"INFO"
"DBUG"
```
or
```
"ERROR"
"WARN "
"INFO "
"DEBUG"
```

Reply:
I'll use the second form.